avocado/plugins/multiplex.py

```diff
@@ -76,8 +72,7 @@ def run(self, args):
         if err:
             log.error(err)
             sys.exit(exit_codes.AVOCADO_FAIL)
-        mux = multiplexer.Mux()
-        root = mux.data
+        mux = args.mux
```

Comment:
Right, this is aligned with what we had in other patches.  The Mux instance will always be in the processed args.